src/adr/0019-distributed-blob-garbage-collector.md

```diff
@@ -0,0 +1,432 @@
+# 19. Distributed blob garbage collector
+
+Date: 2020-02-18
+
+## Status
+
+Proposed
+
+## Context
+
+The body, headers, attachments of the mails are stored as blob on a blob store.
+In order to save space on the those stores, those blobs are de-duplicated using an hash of their content.
+This makes the safe deletion of one of those blob a non trivial problem as we can't delete one blob without ensuring
```

Comment:
a example of a difficult scenario is welcomed also

Reply:
s/those blob/those blobs